src/api/auth/src/authorization.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * JWT Bearer Token Authorization strategy.
+ */
+const passport = require('passport');
+const JwtStrategy = require('passport-jwt').Strategy;
+const { ExtractJwt } = require('passport-jwt');
+
+const { secret, issuer } = require('./token');
+const User = require('./user');
+
+passport.use(
+  new JwtStrategy(
+    {
+      jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+      secretOrKey: secret,
+      issuer,
+    },
+    function (payload, done) {
+      // TODO: check that payload.sub is a user we allow in our system
```

Comment:
Maybe for debugging purposes, should we should add the logger?  (once the TODO has finished)